region/norfair/west.json

```diff
@@ -236,68 +820,193 @@
+                    "Shoot the first Sova to delay its movements."
+                  ]
+                },
+                {
+                  "name": "Leave Sparking",
+                  "notable": false,
+                  "entranceCondition": {
+                    "comeInShinecharging": {
+                      "length": 5,
+                      "openEnd": 1
+                    }
+                  },
+                  "requires": [
+                    "canShinechargeMovement",
+                    {"shinespark": {"frames": 10}}
+                  ],
+                  "exitCondition": {
+                    "leaveWithSpark": {}
+                  }
                 }
               ]
-            }
-          ]
-        },
-        {
-          "from": 5,
-          "to": [
+            },
             {
-              "id": 8,
+              "id": 5,
               "strats": [
                 {
-                  "name": "Base",
+                  "name": "Leave Shinecharged",
                   "notable": false,
-                  "requires": []
+                  "entranceCondition": {
+                    "comeInShinecharging": {
+                      "length": 5,
+                      "openEnd": 1
+                    }
+                  },
+                  "requires": [
+                    "canShinechargeMovement"
+                  ],
+                  "exitCondition": {
+                    "leaveShinecharged": {
+                      "framesRemaining": 15
+                    }
+                  }
                 }
               ]
-            }
-          ]
-        },
-        {
-          "from": 6,
-          "to": [
+            },
+            {
+              "id": 6,
+              "strats": [
+                {
+                  "name": "Leave with Runway",
+                  "notable": false,
+                  "requires": [],
+                  "exitCondition": {
+                    "leaveWithRunway": {
+                      "length": 5,
+                      "openEnd": 1
+                    }
+                  }
+                }
+              ]
+            },
+            {
+              "id": 7,
+              "strats": [
+                {
+                  "name": "Leave Shinecharged",
+                  "notable": false,
+                  "entranceCondition": {
+                    "comeInShinecharging": {
+                      "length": 5,
+                      "openEnd": 0
+                    }
+                  },
+                  "requires": ["canShinechargeMovementComplex"],
+                  "exitCondition": {
+                    "leaveShinecharged": {
+                      "framesRemaining": 30
+                    }
+                  },
+                  "note": "Jump over or kill the Sova."
+                }
+              ]
```

Comment:
Could also add strats to gain a shinecharge in room between 3 and 5.

Reply:
I'll pass on that for now.  14 tiles is hard enough but you also can't touch a transition.